### PR TITLE
Use Github Actions as CI

### LINKED ISSUE #26: Python: Build PyPI package for Windows
We're providing pre-built packages for macOS and Linux, but not for Windows. I consider this a rather significant omission.

Are there any big obstacles that keep us from building, testing, and publishing a Windows package via CircleCI, @normanrz?

### PR DESCRIPTION
**Pull request by normanrz**
This PR adds configuration to build and test the wkw module with Github Actions. Since there is support for Mac, Windows and Linux, we can automate the build and publication of binaries for all major platforms. I think with this we are ready to drop CircleCI for this project.

When merging, I will restrict publication to tags which should resolve the issue with "local versions".

Fixes #26

### CODE REVIEW

File: .github/workflows/python-module.yml [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+  build_win:
+    runs-on: windows-2019
+    steps:
+    - uses: actions/checkout@v1
+    - uses: actions/setup-python@v1
+      with:
+        python-version: '3.x' # Version range or exact version of a Python version to use, using semvers version range syntax.
+        architecture: 'x64' # (x64 or x86)
+    - name: Install lz4
+      shell: bash
```
Comment (philippotto) [inline .github/workflows/python-module.yml]: Is this the linux subsystem bash? Will wkw work in the Windows command line, too? Or isn't this a requirement?

File: .github/workflows/python-module.yml [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+    - name: Test
+      run: |
+        cd python
+        python -c "import wkw"
+        pip install pytest
+        pytest tests
+    - name: Publish
+      if: startsWith(github.event.ref, 'refs/tags')
+      run: |
+        echo "[distutils]" >> $HOME/.pypirc
```
Comment (philippotto) [inline .github/workflows/python-module.yml]: Can this block be extracted into a reusable snippet? Invoking a bash script with this content probably doesn't work, due to the `${{ ... }}` string interpolation :thinking:

File: .github/workflows/python-module.yml (hunk 1 above)
Comment (normanrz) [reply .github/workflows/python-module.yml]: I don't think it is the linux subsystem, but just a bash

Comment (philippotto) [reply .github/workflows/python-module.yml]: I thought, `bash` on windows always refers to the linux-based one. Also commands like `mv` don't exist in the windows cmd as far as I know.

File: .github/workflows/python-module.yml (hunk 2 above)
Comment (normanrz) [reply .github/workflows/python-module.yml]: I now use `twine` instead which uses env vars instead and is the new recommended way of publishing modules.

**Comment by amotta** [comment]
Thanks a lot for that, Norman!

**Pull request merged**